src/server/src/test/java/com/spotify/reaper/resources/CommonToolsTest.java

```diff
@@ -17,14 +33,70 @@ public void testDateTimeToISO8601() {
     DateTime dateTime = new DateTime(2015, 2, 20, 15, 24, 45, DateTimeZone.UTC);
     assertEquals("2015-02-20T15:24:45Z", CommonTools.dateTimeToISO8601(dateTime));
   }
-  
+
   @Test
   public void testParseSeedHost() {
     String seedHostStringList = "127.0.0.1 , 127.0.0.2,  127.0.0.3";
     Set<String> seedHostSet = CommonTools.parseSeedHosts(seedHostStringList);
     Set<String> seedHostExpectedSet = Sets.newHashSet("127.0.0.2","127.0.0.1","127.0.0.3");
-    
+
     assertEquals(seedHostSet, seedHostExpectedSet);
-    
+  }
+
+  @Test
+  public void buildEndpointToRangeMapTest() {
+    Map<List<String>, List<String>> rangeToEndpoint = Maps.newHashMap();
+    rangeToEndpoint.put(Arrays.asList("1","2"), Arrays.asList("node1","node2","node3"));
+    rangeToEndpoint.put(Arrays.asList("2", "4"), Arrays.asList("node1", "node2", "node3"));
+    rangeToEndpoint.put(Arrays.asList("4", "6"), Arrays.asList("node1"));
+    rangeToEndpoint.put(Arrays.asList("6", "8"), Arrays.asList("node1", "node2"));
+
+    Map<String, List<RingRange>> endpointToRangeMap = CommonTools.buildEndpointToRangeMap(rangeToEndpoint);
+
+    assertEquals(endpointToRangeMap.entrySet().size(), 3);
+    assertEquals(endpointToRangeMap.get("node1").size(), 4);
+    assertEquals(endpointToRangeMap.get("node2").size(), 3);
+    assertEquals(endpointToRangeMap.get("node3").size(), 2);
```

Comment:
with `Assert.assertEquals(..)` the expected value is the first parameter.

this is one of the reasonings to use the more elegant and fluid Assert4j library and its `Assertions` class, which is already in the classpath and used in places.